rustls-mio/examples/tlsclient.rs

```diff
@@ -483,7 +483,7 @@ fn make_config(args: &Args) -> Arc<rustls::ClientConfig> {
         let mut reader = BufReader::new(certfile);
         root_store.add_parsable_certificates(&rustls_pemfile::certs(&mut reader).unwrap());
     } else {
-        root_store.add_server_trust_anchors(&webpki_roots::TLS_SERVER_ROOTS);
+        root_store.add_server_trust_anchors(webpki_roots::TLS_SERVER_ROOTS.0);
```

Comment:
Shouldn't we instead just we implement `IntoIterator<Item = OwnedTrustAnchor>` for `TLSClientTrustAnchors` and `TLSServerTrustAnchors` and avoid the need to do the `.0` part?

Reply:
I don't think coherence allows this, since `IntoIterator::Item` is an associated type (not a type param).

Reply:
What's the reason for not exposing a slice directly anyway, but wrapping it in a custom type?